.travis_install.bash

```diff
@@ -81,6 +86,24 @@ case "${TRAVIS_OS_NAME}:${LLVM_CONFIG}" in
     #export PATH=llvmsym/:$PATH
   ;;
 
+  "osx:llvm-config-4.0")
+    brew update
+    brew install shellcheck
+    shellcheck ./.*.bash ./*.bash
+
+    brew install pcre2
+    brew install libressl
+
+    brew install llvm@4.0
+    brew link --overwrite --force llvm@4.0
+    mkdir llvmsym
+    ln -s "$(which llvm-config)" llvmsym/llvm-config-4.0
+    ln -s "$(which clang++)" llvmsym/clang++-4.0
```

Comment:
Would this be OK for the OSX section?

    "osx:llvm-config-4.0")
      brew update
      brew install shellcheck
      shellcheck ./.*.bash ./*.bash

      brew install pcre2
      brew install libressl

      brew install llvm
      # brew link --overwrite --force llvm
      # mkdir llvmsym
      # ln -s "$(which llvm-config)" llvmsym/llvm-config-4.0
      # ln -s "$(which clang++)" llvmsym/clang++-4.0

      # do this elsewhere:
      #export PATH=llvmsym/:$PATH
    ;;